src/gate_preset.cpp

```diff
@@ -36,15 +36,15 @@ void GatePreset::save(nlohmann::json& json, const std::string& section, GSetting
 
   json[section]["gate"]["release"] = g_settings_get_double(settings, "release");
 
-  json[section]["gate"]["curve_threshold"] = g_settings_get_double(settings, "curve-threshold");
+  json[section]["gate"]["curve-threshold"] = g_settings_get_double(settings, "curve-threshold");
 
-  json[section]["gate"]["curve_zone"] = g_settings_get_double(settings, "curve-zone");
+  json[section]["gate"]["curve-zone"] = g_settings_get_double(settings, "curve-zone");
 
-  json[section]["gate"]["hysteresis"] = g_settings_get_boolean(settings, "hysteresis");
+  json[section]["gate"]["hysteresis"] = g_settings_get_boolean(settings, "hysteresis") != 0;
```

Comment:
@Digitalone1 this was the real bug, this would store an integer, because GLib is C and not C++,
and `g_settings_get_boolean()` returns `gboolean` which is `typedef gint   gboolean;`.